akka-cluster/src/main/scala/akka/cluster/Cluster.scala

```diff
@@ -95,11 +100,22 @@ class Cluster(val system: ExtendedActorSystem) extends Extension {
 
   logInfo("Starting up...")
 
+  @Deprecated @deprecated("Access via `clusterTeamInternalFailureDerector` instead", since = "2.5.4")
```

Comment:
interesting that this is public

not sure we have to rename it though, most things are "within the DC" (and when using defaults it's just one DC)